src/commonMain/kotlin/character/classes/rogue/debuffs/DeadlyPoisonDot.kt

```diff
@@ -0,0 +1,61 @@
+package character.classes.rogue.debuffs
+
+import character.Ability
+import character.Debuff
+import data.Constants
+import sim.Event
+import sim.SimParticipant
+import character.Proc
+import character.classes.rogue.talents.*
+
+class DeadlyPoisonDot(owner: SimParticipant) : Debuff(owner) {
+    companion object {
+        const val name = "Deadly Poison (DoT)"
+    }
+
+    override val name: String = Companion.name
+    override val durationMs: Int = 12000
+    override val tickDeltaMs: Int = 3000
+    val totalTicks = durationMs / tickDeltaMs
+    override val maxStacks: Int = 4
```

Comment:
Right, got it confused because it starts at 0, but `refresh()` only checks for `<` not `<=`

Reply:
Actually nevermind, I got confused by how a buff is applied in `SimParticipant.addDebuff()`. `Buff.reset()` sets it to 0, but since `Buff.refresh()` is called on every buff, it's set to 1 always. Still confusing because for non-stacking buffs/debuffs the `currentStacks` will always be 0 instead of a more logical 1 (also means "AvgStacks" shows 0 in the stats).